miniforge-cuda/Dockerfile

```diff
@@ -91,11 +91,12 @@ RUN wget --quiet ${MINIFORGE_URL} -O /miniforge.sh \
     && echo ". /opt/conda/etc/profile.d/conda.sh" >> /etc/skel/.bashrc \
     && echo "auto_update_conda: False" >> /opt/conda/.condarc \
     && echo "ssl_verify: False" >> /opt/conda/.condarc \
-    && chmod -R ugo+w /opt/conda \
     && ln -s /opt/conda /conda
 
 # Install tini for init
 RUN conda install -k -y tini
 
+RUN chmod -R ugo+w /opt/conda
```

Comment:
After reviewing again I think we should move L87 to this section

```
/opt/conda/bin/conda clean -tipsy
``` 
to here so we can clean up after the `tini` install

```suggestion
# Clean up conda and set permissions for all users
RUN /opt/conda/bin/conda clean -tipsy \
    && chmod -R ugo+w /opt/conda
```

Reply:
Done